### ISSUE TITLE
New Site

### ISSUE THREAD

**Issue by Dr4mEdo** [briefbanane/manga-downloader#7]
Hello, thanks for the script, it's very usefull and quite easy to use.

I would like to know if it's possible to add http://www.japscan.com to the supported website list.

Thanks in advance and keep up the good work !

**Comment by briefbanane** [comment]
I looked into it and everything seemed nice, until I checked on the individual images. As you can see here: http://cdn.japscan.com/cr-images/Naruto/423/01.jpg they saved there images in a non human-readable way. They use some Javascript to reposition the individual pieces when you view it in your Browser. If somebody would provide a function that fixes the image, supporting that site would be no problem. Maybe I'll look into it, but I'm not keen on doing it.

**Comment by Dr4mEdo** [comment]
Thanks for your reply.
That really too bad that we can't use the image as it is^^.
Maybe someone will have time to look into it.

**Comment by briefbanane** [comment]
b14afa8cfb73a776431b6986ae13edb167db0103 Well I think I did it. I figured out how the images are scrambled and since all the images are scrambled in the same way, it wasn't too hard. At least with the help of ImageMagick ;) Make sure you have that installed (I think most distros have it pre-installed) and if other problems occur (I only tested the script with a few manga) feel free to open another issue.
